src/routes/tags.rs

```diff
@@ -93,9 +93,6 @@ pub async fn loader_list(pool: web::Data<PgPool>) -> Result<HttpResponse, ApiErr
     Ok(HttpResponse::Ok().json(results))
 }
 
-// At some point this may take more info, but it should be able to
-// remain idempotent
-// TODO: don't fail if loader already exists
```

Comment:
Still an issue (Although I didn't put a comment directly on this one, it was still mentioned in the previous comments).